core/src/test/scala/com/rallyhealth/vapors/core/example/JoeSchmoe.scala

```diff
@@ -9,7 +9,7 @@ object JoeSchmoe {
   val age = FactTypes.Age(32)
   val userRole = FactTypes.Role(Role.User)
   val adminRole = FactTypes.Role(Role.Admin)
-  val dateOfBirth = FactTypes.DateOfBirth(LocalDate.of(1987, 1, 1))
+  val dateOfBirth = FactTypes.DateOfBirth(LocalDate.of(1988, 8, 8))
```

Comment:
This wasn't needed, but it seems easier to remember 8/8/88 and has a reasonable year in the past.